eslint.config.js

```diff
@@ -0,0 +1,221 @@
+'use strict'
+
+const globals = require('globals')
+const { FlatCompat } = require('@eslint/eslintrc')
+
+const eslintrc = new FlatCompat({
+  baseDirectory: __dirname
+})
+
+module.exports = [
+  {
+    ignores: [
+      '.nyc_output',
+      'coverage',
+      'node_modules',
+      'tests/fixtures',
+      'tests/integrations/eslint-plugin-import',
+
+      '!.vitepress',
+      'docs/.vitepress/dist',
+      'docs/.vitepress/build-system/shim/eslint.mjs',
+      'docs/.vitepress/build-system/shim/assert.mjs',
+      'docs/.vitepress/build-system/shim/path.mjs',
```

Comment:
This line was not in `.eslintrc.js` but I was getting linting errors in this file with flat config, so I added this here. I'm unsure why flat config was showing errors on this file while eslintrc was not, but please double-check that this should be excluded.